modules/ROOT/pages/provisioning-gcp.adoc

```diff
@@ -71,3 +71,10 @@ gcloud compute instances create --metadata-from-file "user-data=${CONFIG}" --ima
 ----
 
 NOTE: By design, https://cloud.google.com/compute/docs/startupscript[startup scripts] are not supported on FCOS. Instead, it is recommended to encode any startup logic as systemd service units in the Ignition configuration.
+
+=== Ignition via the GCP console
```

Comment:
How about instead of making this a separate section, we rework the sentence a bit above to say:

> In order to launch a customized FCOS instance, a valid Ignition configuration must be passed as metadata under the `user-data` key at creation time. In the web console, this is available under the Management section. From the command-line, use `--metadata-from-file`:

Reply:
Thanks, indeed that's better.  I made some more tweaks.